### PR TITLE
Added Eth Guides template and beginning of IPFS eth Guide

### PR DESCRIPTION
**Pull request by walkerlj0**

### CODE REVIEW

**Comment by walkerlj0** [comment]
@MarcoPolo @enolal826 take a look at the template for the eth guides, and when done approve! Last approver please merge

File: ethguides/intro-ipfs-filecoin.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+id: ?
+title: Introduciton to IPFS and Filecoin
```
Comment (enolal826) [inline ethguides/intro-ipfs-filecoin.md:3]:
```suggestion
title: Introduction to IPFS and Filecoin
```

File: ethguides/intro-ipfs-filecoin.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+date: 'July 24, 2022'
+slug: ipfs-101
+description: Conceptual understanding of they why and what of IPFS and FIlecoin
```
Comment (enolal826) [inline ethguides/intro-ipfs-filecoin.md:6]:
```suggestion
description: Conceptual understanding of the why and what of IPFS and FIlecoin
```

File: ethguides/intro-ipfs-filecoin.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+tags: ['101', 'web3']
+author: ProtocolLabs
+avatar: ipfs-logo.jpg
+---
+<!-- Note: Above header will be changed to a JSON file -->
+
+<Section name="1. Why were IPFS and Filecoin Created?">
+
+## Properties of the Next Version of the Internet
+> _The Properties of the internet and it's technologies define our human rights_ Juan Benet 2022
```
Comment (enolal826) [inline ethguides/intro-ipfs-filecoin.md:18]: I think it should be
```suggestion
> _The Properties of the internet and its technologies define our human rights_ Juan Benet 2022
```

File: ethguides/intro-ipfs-filecoin.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Technologies like household automation have given women independence, and the internet has given mass accessibility to education, ideas, and collaboration on a global scale.
+
+Protocol Labs and it's network oragnization's goal is, through the development of tecnologies, to
```
Comment (enolal826) [inline ethguides/intro-ipfs-filecoin.md:22]:
```suggestion
Protocol Labs and its network organization's goal is, through the development of tecnologies, to
```

**Review by enolal826 — APPROVED** [review]
Just left comments for a few typos

**Pull request merged**